docker/lwua-ingest/lwua-py/tests/test_graphdb.py

```diff
@@ -0,0 +1,67 @@
+import pytest
+from datetime import datetime
+from pathlib import Path
+from lwua.graphdb import (
+    context_2_fname,
+    suffix_2_format,
+    read_graph,
+    convert_results_registry_of_lastmod,
+)  # replace 'your_module_path' with the actual module path
+from lwua.ingest import data_path_from_config
+
+results = {
```

Comment:
yuk - sparql details all over this struct -- don't see why this is part of any sensible functional test?